src/components/App.js

```diff
@@ -0,0 +1,10 @@
+import BooksListContainer from '../containers/BooksList';
+
+const App = () => (
+  <div className="App">
+    <BooksListContainer />
+    {/* <BooksForm /> */}
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.
- [ ] <b>REQUIRED:</b> Project requires this `BooksForm` to be here. Kindly remove the comments and make sure that it stays there.